subprojects/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -221,6 +233,42 @@ private static void validatePackageName(String packageName) {
         }
     }
 
+    static class JavaVersionSelection {
```

Comment:
Instead of Java, it might be better for this to always ask about JDKs.  Saying Java is slightly confusing for people creating Groovy or Kotlin projects.